pyoverkiz/client.py

```diff
@@ -629,8 +633,8 @@ async def __get(self, path: str) -> Any:
         """Make a GET request to the OverKiz API"""
         headers = {}
 
-        if self.server == SUPPORTED_SERVERS["somfy_europe"]:
-            await self._refresh_somfy_tahoma_token_if_expired()
+        await self._refresh_token_if_expired()
```

Comment:
Just a double check, here we call this before we check if there is an access token. In post we call this after we checked if we have an access token.

Reply:
That’s because in the `__post`, I don’t want this method to be call for the `login` endpoint. It’s not about the access_token.